src/Custom/Audio/OpenAIAudioModelFactory.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenAI.Audio;
+
+/// <summary> Model factory for models. </summary>
+public static partial class OpenAIAudioModelFactory
+{
+    /// <summary> Initializes a new instance of <see cref="OpenAI.Audio.AudioTranscription"/>. </summary>
+    /// <param name="duration"> The duration of the input audio. </param>
+    /// <returns> A new <see cref="OpenAI.Audio.AudioTranscription"/> instance for mocking. </returns>
+    public static AudioTranscription AudioTranscription(string language = null, TimeSpan? duration = null, string text = null, IEnumerable<TranscribedWord> words = null, IEnumerable<TranscribedSegment> segments = null)
+    {
+        words ??= new List<TranscribedWord>();
+        segments ??= new List<TranscribedSegment>();
+
+        return new AudioTranscription(
+            task: default,
```

Comment:
It means the internal members will be set to their default values. In this case, the internal value will be `null`. I can set it to `InternalCreateTranscriptionResponseVerboseJsonTask.Transcribe` if you think we may eventually benefit from that. I didn't bother doing it because it was an internal property that didn't seem to have any particular use.

Reply:
I think for "correctness" and because it's a simple change, let's set it to `Transcribe`. It's true that it is an internal property, but it is technically visible if the mock object is serialized.